proposals/2021-11-ibc-patch-upgrade/proposal.json

```diff
@@ -0,0 +1,4 @@
+{
+    "title":"IBC Patch Upgrade",
+    "description":"The recent upgrade to Regen Ledger v2 was a great success. However there's a bug in the integration, which created an issue for IBC transfers. IBC transfers are not going through currently due to a missing migration for the IBC module. Regen Ledger v2.1.0 fixes that and this proposal is to conduct an emergency upgrade to avoid any further potential issues with IBC client expiry. If IBC clients expire, the funds would get stuck in the expired channel, so we need to upgrade the network to fix this before IBC clients expire.\n\nThis proposal expects every validator to update their regen software to v2.1.0 before the block height 3126318 and then to vote on this proposal. A yes vote on this proposal conveys that the validator has updated their binaries to v2.1.0.\n\nEstimated update time is Friday, 26th Nov, 1700UTC.\n\nMore details are available in the long-form proposal: https://github.com/regen-network/governance/tree/main/proposals/2021-11-ibc-patch-upgrade"
```

Comment:
```suggestion
    "description":"The recent upgrade to Regen Ledger v2 was a great success. However there's a bug in the integration, which created an issue for IBC transfers. IBC transfers are not going through currently due to a missing migration for the IBC module. Regen Ledger v2.1.0 fixes that and this proposal is to conduct an emergency upgrade to avoid any further potential issues with IBC client expiry. If IBC clients expire, the funds would get stuck in the expired channel, so we need to upgrade the network to fix this before IBC clients expire.\n\nThis proposal expects every validator to update their regen software to v2.1.0 before the block height 3126912 and then to vote on this proposal. A yes vote on this proposal conveys that the validator has updated their binaries to v2.1.0.\n\nEstimated update time is Friday, 26th Nov, 1700UTC.\n\nMore details are available in the long-form proposal: https://github.com/regen-network/governance/tree/main/proposals/2021-11-ibc-patch-upgrade"
```